src/us/kbase/workspace/database/mongo/MongoWorkspaceDB.java

```diff
@@ -699,86 +699,67 @@ private void setCreatedWorkspacePermissions(
 	private static final Set<String> FLDS_WS_META = newHashSet(Fields.WS_META);
 
 	@Override
-	public Instant setWorkspaceMeta(final ResolvedWorkspaceID rwsi,
+	public Instant setWorkspaceMeta(
+			final ResolvedWorkspaceID rwsi,
 			final WorkspaceUserMetadata newMeta)
-			throws WorkspaceCommunicationException,
-			CorruptWorkspaceDBException {
-
+			throws WorkspaceCommunicationException, CorruptWorkspaceDBException {
+		// TODO NOW CODE integrate keys to remove (below) into this method and do it all in
+		// 					one shot
```

Comment:
mysterious space?

Reply:
I tabbed it in to make it clear it's a continuation of the TODO